source/entities/EntityBase.h

```diff
@@ -127,6 +128,15 @@ class EntityBase
     { return false; }
     virtual void drop(const Ogre::Vector3& v)
     {}
+    inline void show()
+    {
+        RenderManager::getSingleton().rrAttachEntity(this);
```

Comment:
No worries, you won't see the game speed difference. And if this call is called so much that inlining it would change something, it will be time to inline it.
Trying to have readable code if the first goal we should have when coding. Performances will come after (where needed). The real time wasters are almost never where you expect them.